.goreleaser.yml

```diff
@@ -23,6 +23,18 @@ builds:
       - amd64
 archives:
   - format: tar.gz
+brews:
+  -
+    github:
+      owner: vapor-ware
+      name: homebrew-formula
+    commit_author:
+      name: vio-bot
+      email: 'bot@vapor.io'
```

Comment:
note: I don't know what the bot email is, so I'm pretty sure this will need to change